docs/wallet-integration.md

```diff
@@ -0,0 +1,126 @@
+## Wallet Integration
+
+This guide explains how to integrate wallets with the Babylon staking dApp. The
+dApp supports both Bitcoin (BTC) and Babylon (BBN) wallets through two
+integration paths:
+
+### 1. Integration Through Tomo Connect SDK Lite (Recommended)
+
+The recommended way to integrate your wallet with Babylon staking dApp is
+through
+[Tomo Connect SDK Lite](https://docs.tomo.inc/tomo-sdk/tomo-connect-sdk-lite).
+This provides:
+
+- Simplified integration process
+- Broader wallet ecosystem access
+- Maintained by Tomo team
```

Comment:
We don't have to sell it. We are stating what users should do and we should be clear why we recommend users to work with Tomo Connect: Native integrations are maintained by us and we are only going to do it for critical wallets.